CMakeLists.txt

```diff
@@ -44,6 +44,11 @@ endif()
 
 FetchContent_MakeAvailable(GraphStreamingCC)
 
+# hack to re-use GraphStreamingCC boost installation by hardcoding BOOST_ROOT
+set(BOOST_ROOT "${CMAKE_CURRENT_BINARY_DIR}/GraphStreamingCC/src/GraphStreamingCC-build/boost-src")
```

Comment:
is this still necessary now that we don't use boost?

Reply:
It's necessary for the AGM sketching performance test unfortunately.